rhel7/profiles/stig-rhel7-disa.xml

```diff
@@ -523,7 +523,9 @@ Storage deployments.
 <select idref="audit_rules_privileged_commands_ssh_keysign" selected="true" />
 
 <!-- SRG-OS-000042-GPOS-00020, SV-86805r2_rule, RHEL-07-030790 -->
-<select idref="audit_rules_privileged_commands_pt_chown" selected="true" />
+<!-- No longer built as part of RHEL7. See 
+     https://bugzilla.redhat.com/show_bug.cgi?id=984828 for more info 
```

Comment:
Please change bugzilla link to public one :)

Reply:
@dahaic done